etc/mock_data_dir/cat1/fuction_select/pretty_count_star.json

```diff
@@ -0,0 +1,21 @@
+{
+  "results": [
+    {
+      "series": [
+        {
+          "name": "average_temperature",
+          "columns": [
+            "time",
+            "count_degrees"
+          ],
+          "values": [
+            [
+              0,
+              15258
+            ]
+          ]
+        }
+      ]
+    }
+  ]
+}
```

Comment:
everything in the `/mock_data_dir` directory was just meant to be for testing that walking the directory structure works correctly.